deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -217,23 +220,24 @@ data:
         </buffer>
       </match>
     </label>
-  logs.source.systemd.conf: |-
+  logs.source.kubelet.conf: |-
     <match host.kubelet.**>
       @type relabel
       @label @KUBELET
     </match>
     <label @KUBELET>
       <filter host.kubelet.**>
         @type kubernetes_sumologic
-        source_category kubelet
-        source_name k8s_kubelet
-        source_category_prefix "#{ENV['SOURCE_CATEGORY_PREFIX']}"
-        exclude_facility_regex "#{ENV['EXCLUDE_FACILITY_REGEX']}"
-        exclude_host_regex "#{ENV['EXCLUDE_HOST_REGEX']}"
-        exclude_priority_regex "#{ENV['EXCLUDE_PRIORITY_REGEX']}"
-        exclude_unit_regex "#{ENV['EXCLUDE_UNIT_REGEX']}"
+        source_category "kubelet"
```

Comment:
there were no quotes before for this.

Reply:
These values are parsed as string, and the quotes are removed when taking the value into account